packages/backend-output-schemas/API.md

```diff
@@ -329,6 +335,26 @@ export const unifiedBackendOutputSchema: z.ZodObject<{
             customOutputs: string;
         };
     }>]>>;
+    "AWS::Amplify::Function": z.ZodOptional<z.ZodDiscriminatedUnion<"version", [z.ZodObject<{
+        version: z.ZodLiteral<"1">;
+        payload: z.ZodObject<{
+            customerFunctions: z.ZodString;

```

Comment:
Should we use `customer` in property naming?
Perhaps `definedFunctions` (to follow `defineFunction` naming?)

Reply:
I used `customer` as opposed to functions made by `Amplify`. I still think functions made by Amplify can still be considered as `definedFunctions` which may lead to confusion.

Reply:
One of the reasons I made this comment is that screenshot in PR description.
where deployment log shows `customerFunctions` - de facto making this name customer facing.
`customer` is terminology that we use and understand, not sure about our customers.


> I still think functions made by Amplify can still be considered as definedFunctions which may lead to confusion.

I can see that and I can't produce better suggestion.

@edwardfoyle @Amplifiyer maybe you guys would have good suggestion for a name?

Reply:
`backendFunctions`?

Reply:
I think `definedFunctions` is a pretty good name. Maybe `exposedFunctions` or `exportedFunctions` as other options?

Reply:
I'm thinking maybe we can keep `definedFunctions` for now since, from a customer's POV this should be the list of functions they defined with `defineFunction`.

My only concern is if they look at their function stack and wonder why the `Amplify` owned functions aren't listed there since "defined" in this case means functions from `defineFunction` and not "defined" in CFN function stack. This may be a needless worry.

Reply:
> why the Amplify owned functions aren't listed there since "defined"

It's like private members of a class. They're implementation details and shouldn't concern the caller. I.e. they are not visible from user pov.

Reply:
They are visible in CFN, but I don't think we have to worry about customers seeing them. Maybe it's safe to assume they won't see it unless they are curious or trying to debug and going through CFN.